docs/getting_started/features/metadata.md

```diff
@@ -6,15 +6,47 @@ GTFS feeds can contain further information regarding the feed itself and its pro
 
 Feed Information communicates important information about the feed, such as its validity (start day and end date), the publishing organization, and contact information for inquiries regarding the GTFS dataset and data publishing practices.  
 
+**Pre-requirements**: Implement Base features.
+
 | Files included                   | Fields included   |
 |----------------------------------|-------------------|
-|[feed_info.txt](/schedule/reference/#feed_infotxt)|`feed_publisher_name`, `feed_publisher_url`, `feed_lang`, `default_lang`, `feed_start_date`, `feed_end_date`, `feed_version`, `feed_contact_email`, `feed_contact_url` |
+|[feed_info.txt](/documentation/schedule/reference/#feed_infotxt)|`feed_publisher_name`, `feed_publisher_url`, `feed_lang`, `default_lang`, `feed_start_date`, `feed_end_date`, `feed_version`, `feed_contact_email`, `feed_contact_url` |
+
+??? note "Sample Data"
+    <p style="font-size:16px">
+    The following table shows additional information relative to the GTFS dataset, including its validity, version and contact information.
+    </p>
+    !!! note ""
+        <p style="font-size:16px">
+        <a href="/documentation/schedule/reference/#feed_infotxt"><b>feed_info.txt</b></a> <br>
+        </p>
 
+        | feed_publisher_name	      | feed_publisher_url   | feed_lang | default_lang | feed_start_date | feed_end_date | feed_version | feed_contact_email | feed_contact_url             |
+        |--------------------------|----------------------|-----------|--------------|-----------------|---------------|--------------|--------------------|------------------------------|
+        | Greater Region Transport | https://www.gra1.org | en        | en           |        20240101 |      20241231 |          3.1 | support@gra1.org   | https://www.gra1.org/support |
 
 ## Attributions
 
 Attributions makes it possible to share additional details regarding the organizations involved in the creation of the dataset (producers, operators and/or authorities, etc.).
 
+**Pre-requirements**: Implement Base features.
+
 | Files included                   | Fields included   |
 |----------------------------------|-------------------|
-|[attributions.txt](/schedule/reference/#attributionstxt) |`attribution_id`, `agency_id`, `route_id`, `trip_id`, `organization_name`, `is_producer`, `is_operator`, `is_authority`, `attribution_url`, `attribution_email`, `attribution_phone` |
+|[attributions.txt](/documentation/schedule/reference/#attributionstxt) |`attribution_id`, `agency_id`, `route_id`, `trip_id`, `organization_name`, `is_producer`, `is_operator`, `is_authority`, `attribution_url`, `attribution_email`, `attribution_phone` |
+
+??? note "Sample Data"
+    <p style="font-size:16px">
+    The following table shows additional information that specifies the roles that multiple organizations play relative to the GTFS dataset, such as data producers, operators and transit authorities, providing also their corresponding contact information. 
```

Comment:
Same comment, this is a repetition of the above description. It looks like the sample is self-explanatory, we can remove this description IMO.